src/py/flwr/server/app.py

```diff
@@ -388,6 +417,59 @@ def run_superlink() -> None:
         driver_server.wait_for_termination(timeout=1)
 
 
+def _try_setup_client_authentication(
+    args: argparse.Namespace,
+    certificates: Optional[Tuple[bytes, bytes, bytes]],
+) -> Optional[Tuple[Set[bytes], ec.EllipticCurvePublicKey, ec.EllipticCurvePrivateKey]]:
+    if args.require_client_authentication:
+        if certificates is None:
```

Comment:
Are thery any specific formats we support (like PEM)?
Are we able or plan to implement this only with SSH or also with SSL-Certificates?

1. Check if Priv and Public Key are in the respective valid format -> [Maybe this helps](https://gist.github.com/uilianries/0459f59287bd63e49b1b8ef03b30d421)
2. Verify that Cert and Priv Key are belonging together ->  [Maybe this helps](https://stackoverflow.com/questions/69167192/verifying-private-key-belongs-to-certificate-with-pyca)

Reply:
We support OpenSSH format for the key pair. It's checked by cryptography library for validity.